test/k8s-integration/main.go

```diff
@@ -479,7 +479,7 @@ func handle() error {
 	}
 
 	skipDiskImageSnapshots := false
-	if mustParseVersion(testParams.clusterVersion).lessThan(mustParseVersion("1.22.0")) {
+	if mustParseVersion(testParams.clusterVersion).lessThan(mustParseVersion("1.22.0")) || strings.Contains(testParams.storageClassFile, "sc-extreme") {
 		// Disk image cloning in only supported from 1.22 on.
```

Comment:
I don't think we need to add the skip for extreme, if we remove the capability in driver-config.go?

Reply:
This one is not doing anything, let me delete this. Initially i was thinking at submit we will revert the last commit [Test pd-extreme k8s integration](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1079/commits/252c7d29dfa9c817f827e82c17a7bcf78e152861) but we actually needed to submit it. Will modify the code